bindings/nodejs/lib/types/block/output/unlock-condition.ts

```diff
@@ -156,8 +156,8 @@ class TimelockUnlockCondition extends UnlockCondition {
 }
 
 /**
- * Defines an expiration slot index. Before the slot index is reached, only the Address defined in the Address
- * Unlock Condition is allowed to unlock the output. Afterward, only the Return Address can unlock it.
+ * Defines a slot index until which only Address, defined in Address Unlock Condition, is allowed to unlock the output.
+ * After the slot index is reached/passed, only Return Address can unlock it.
```

Comment:
```suggestion
 * Defines an unlock condition that binds an expiry time/slot index to the address given in the Address Unlock 
 * Condition of the same output. Once that address has expired, only the address defined in `returnAddress` can 
 * unlock the output. 
```

Reply:
i suggested the current line in the tip document. unless you feel like your suggestion is better?